jest/moduleNameMapper.js

```diff
@@ -1,86 +1,58 @@
 const path = require('path');
 
 const { pathsToModuleNameMapper } = require('ts-jest');
-const {
-  sys,
-  findConfigFile,
-  readConfigFile,
-  parseJsonConfigFileContent,
-} = require('typescript');
 
-/**
- * Set a default `src` module alias for backward compatibility.
- *
- * TODO: drop this default in skuba v5.
- */
-const DEFAULT_PATHS = { src: ['src'], 'src/*': ['src/*'] };
+const { tryParseTsConfig } = require('./tsConfig');
 
-/**
- * @returns {unknown}
- */
-const getConfigFromDisk = () => {
-  const filename =
-    findConfigFile('.', sys.fileExists.bind(this)) ?? 'tsconfig.json';
+module.exports.createModuleNameMapper = (getConfig) => {
```

Comment:
Sorry for the big diff here, functional changes should be:

1. Moving the try-catch to shared function in `jest/tsConfig.js`
2. Defaulting to no paths rather than `src`